### DISCUSSION TITLE
Allow relative url with https protocol in bleach

### DISCUSSION THREAD

**Discussion by LocNguyen-KMS** [mozilla/bleach#661] [Q&A] [answered]
This is what I got for bleach==2.0.0
<img width="721" alt="Screen Shot 2022-05-09 at 11 21 25" src="https://user-images.githubusercontent.com/98076735/167374370-2e183f9a-9034-4d64-8910-3ff5207e8e97.png">
In bleach version 2, the relative url is not removed. Is is possible for me to keep this relative url in bleach version >= 3?

This is what I got for bleach==3.3.1
<img width="721" alt="Screen Shot 2022-05-09 at 11 14 27" src="https://user-images.githubusercontent.com/98076735/167374529-facd5879-d4be-442a-9cc4-a1f64401946f.png">

**Comment by willkg** [comment, marked as answer]
This shouldn't be a question--this should be an issue because it's describing a bug. Can you file this as an issue? Thank you!

**Reply by LocNguyen-KMS** [reply]
sure, thank you. I have just raised an issue about this
